docs/how-to/cooperative_groups.rst

```diff
@@ -0,0 +1,403 @@
+-----------------------------
+
+This section describes how to synchronize the group over multiple GPUs:
+
+1. Confirm the cooperative launch capability over multiple GPUs:
+
+.. code-block:: C++
+  
+  // Check support of cooperative groups
+  std::vector<int> deviceIDs;
+  for(int deviceID = 0; deviceID < device_count; deviceID++) {
+  #ifdef __HIP_PLATFORM_AMD__
+    int supports_coop_launch = 0;
+    HIP_CHECK(
+      hipDeviceGetAttribute(
+        &supports_coop_launch, 
+        hipDeviceAttributeCooperativeMultiDeviceLaunch, 
+        deviceID));
+    if(!supports_coop_launch) {
+      std::cout << "Skipping, device " << deviceID << " does not support cooperative groups"
+                << std::endl;
+    }
+    else
+  #endif
+    {
+      std::cout << deviceID << std::endl;
+      // Collect valid deviceIDs.
+      deviceIDs.push_back(deviceID);
+    }
+  }
+
+  if(!deviceIDs.size())
+  {
+      std::cout << "No valid GPU found." << std::endl;
+  } else {
+        std::cout << "Valid GPUs number:" << deviceIDs.size() << std::endl;
+  }
+
+2. Launch the cooperative kernel over multiple GPUs:
+
+.. code-block:: C++
+
+    hipLaunchParams *launchParamsList = (hipLaunchParams*)malloc(sizeof(hipLaunchParams) * deviceIDs.size());
+    for(int deviceID : deviceIDs) {
+		// Set device
+		HIP_CHECK(hipSetDevice(deviceID));
+		
+		// Create stream
+        hipStream_t stream;
+        HIP_CHECK(hipStreamCreate(&stream));
+		
+		// Parameters
+		void* params[] = {&(d_vector[deviceID]), &(d_block_reduced[deviceID]), &(d_partition_reduced[deviceID])};
+		
+		// Set launchParams
+        launchParamsList[deviceID].func = (void*)vector_reduce_kernel<partition_size>;
+        launchParamsList[deviceID].gridDim = dim3(1);
+        launchParamsList[deviceID].blockDim = dim3(threads_per_block);
+        launchParamsList[deviceID].sharedMem = 0;
+        launchParamsList[deviceID].stream = stream;
+        launchParamsList[deviceID].args = params;
+    }
+
+    HIP_CHECK(hipLaunchCooperativeKernelMultiDevice(launchParamsList,
+                                                    (int)deviceIDs.size(),
+                                                    hipCooperativeLaunchMultiDeviceNoPreSync));
+
+3. The device side code of the grid synchronization over multiple GPUs:
+
+.. code-block:: C++
+
+  multi_grid_group multi_grid = this_multi_grid();
+  multi_grid.sync();
+
+Unsupported NVIDIA CUDA features
+===========================
+
+HIP doesn't support the following NVIDIA CUDA optional headers:
+
+.. code-block:: C++
+
+    // Optionally include for memcpy_async() collective
+    #include <cooperative_groups/memcpy_async.h>
+    // Optionally include for reduce() collective
+    #include <cooperative_groups/reduce.h>
+    // Optionally include for inclusive_scan() and exclusive_scan() collectives
+    #include <cooperative_groups/scan.h>
+
+HIP doesn't support the following CUDA class the in ``cooperative_groups`` namespace:
+
+* ``cluster_group``
+
```

Comment:
Something is messed up here. 
Perhaps the second highlighted line should be deleted. 
![image](https://github.com/ROCm/HIP/assets/164940351/a5b29d82-9643-45db-a4ec-3617d9960b2d)

Reply:
Fixed.